ReactAndroid/src/main/jni/react/jni/AndroidJSCFactory.h

```diff
@@ -10,7 +10,7 @@
 
 namespace folly {
 
-class dynamic;
+struct dynamic;
```

Comment:
This is a struct not class see folly sources.
clang is more diligent about such warnings.